### PR TITLE
Fixed OAuth flow test for new Nuts Node

### PR DESCRIPTION
**Pull request by reinkrul**

### CODE REVIEW

File: oauth-flow/run-test.sh [hunk 1]
```diff
@@ -23,61 +19,41 @@ echo "------------------------------------"
 echo "Registering vendors..."
 echo "------------------------------------"
 # Register Vendor A
-docker-compose exec -e NUTS_MODE=cli nodeA-backend nuts crypto selfsign-vendor-cert "Vendor A" /opt/nuts/keys/vendor_certificate.pem
-docker-compose exec -e NUTS_MODE=cli nodeA-backend nuts registry register-vendor /opt/nuts/keys/vendor_certificate.pem
+VENDOR_A_DIDDOC=$(docker-compose exec nodeA-backend nuts vdr create-did)
+VENDOR_A_DID=$(echo $VENDOR_A_DIDDOC | jq -r .id)
```
Comment (woutslakhorst) [inline oauth-flow/run-test.sh:23]: add `jq` as development dependency

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: oauth-flow/run-test.sh (hunk 1 above)
Comment (woutslakhorst) [reply oauth-flow/run-test.sh:23]: add to docs

**Review by woutslakhorst — APPROVED** [review]

File: oauth-flow/docker-compose.yml [hunk 2]
```diff
@@ -1,34 +1,38 @@
 version: "3.7"
 services:
   nodeA-backend:
```
Comment (stevenvegt) [inline oauth-flow/docker-compose.yml:3]: I think it is confusing that nodeA is an nginx container while nodeB is the nuts-node where nodeA-backend it the nuts-node for A.

**Review by stevenvegt — CHANGES_REQUESTED** [review]

File: oauth-flow/docker-compose.yml (hunk 2 above)
Comment (reinkrul) [reply oauth-flow/docker-compose.yml:3]: Might be, but out of scope for this PR

**Pull request merged**